core/src/main/scala/org/apache/spark/storage/BlockManagerDecommissioner.scala

```diff
@@ -33,7 +33,7 @@ import org.apache.spark.util.ThreadUtils
 
 /**
  * Class to handle block manager decommissioning retries.
- * It creates a Thread to retry offloading all RDD cache and Shuffle blocks
+ * It creates a Thread to retry migrating all RDD cache and Shuffle blocks
```

Comment:
There are a few more places where offload is used in the `BlockManagerSuite`:
- https://github.com/apache/spark/blob/d562aaf4ff350b66aef354ca4493c60e75d2b04b/core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala#L1895
- https://github.com/apache/spark/blob/d562aaf4ff350b66aef354ca4493c60e75d2b04b/core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala#L1913
- https://github.com/apache/spark/blob/d562aaf4ff350b66aef354ca4493c60e75d2b04b/core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala#L1929

- https://github.com/apache/spark/blob/d562aaf4ff350b66aef354ca4493c60e75d2b04b/core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala#L1931